android/app/src/main/java/com/reactnativebase/MainApplication.java

```diff
@@ -3,6 +3,7 @@
 import android.app.Application;
 
 import com.facebook.react.ReactApplication;
+import com.reactnativecommunity.asyncstorage.AsyncStoragePackage;
```

Comment:
You need to instanciate this package. 
https://github.com/react-native-community/async-storage/blob/LEGACY/docs/Linking.md
We should also check if another imported package is not being used.